packages/feedback/src/core/sendFeedback.ts

```diff
@@ -35,7 +36,10 @@ export const sendFeedback: SendFeedback = (
   // We want to wait for the feedback to be sent (or not)
   return new Promise<string>((resolve, reject) => {
     // After 5s, we want to clear anyhow
-    const timeout = setTimeout(() => reject('Unable to determine if Feedback was correctly sent.'), 5_000);
+
+    const timeout = setTimeout(() => {
+      reject(new Error('Unable to determine if Feedback was correctly sent.'));
```

Comment:
This updates the thrown type to be `Error` instead of `string` so that users can consistently check & optionally render error values.

Now all problems return as Error objects, and can be used like:
```
try {
  await sendFeedback(...);
} catch (error) {
  console.log(error.message); // instead of: `'message' in error ? error.message : error`
}
```

Reply:
Hmm, will this break for us (+ existing users) that expect the rejection to be a string? I think we use the rejected value directly as the error message in our error toast.

Reply:
I removed the return/error type changes.